content/src/content/jcr_root/apps/core/wcm/sandbox/components/teaser/v1/teaser/_cq_dialog/.content.xml

```diff
@@ -22,26 +22,26 @@
                             sling:resourceType="granite/ui/components/coral/foundation/container"
                             margin="{Boolean}true">
                         <items jcr:primaryType="nt:unstructured">
-                            <withCTA
+                            <actionsEnabled
                                     jcr:primaryType="nt:unstructured"
                                     sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                    fieldDescription="Check to allow adding Call To Action (CTA) elements."
+                                    fieldDescription="Check to allow adding Call To Action elements."
                                     text="With Call-To-Action(s)"
-                                    name="./withCTA"
+                                    name="./actionsEnabled"
                                     value="{Boolean}true"
                                     uncheckedValue="{Boolean}false"
-                                    granite:hide="${cqDesign.disableCTA}">
-                            </withCTA>
-                            <ctas
+                                    granite:hide="${cqDesign.disabled}">
+                            </actionsEnabled>
+                            <actions
                                     jcr:primaryType="nt:unstructured"
                                     sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
```

Comment:
@vladbailescu, @andreeadracea - I see a few UX issues with the teaser dialog.

* Actions multifield
  * It's not easy to distinguish each action - there's just a vertical list of text inputs with equal weight. _proposal:_ Group the action inputs better, perhaps putting the link and text inputs on one line (see how we did this with the style system policy dialog).
  * It takes up a lot of vertical space and can expand downwards, pushing the other key options out of view. _proposal:_ Move the multifield to its own "Actions" tab which gets shown / hidden when selecting "with Call-To-Action(s)".
* Link
  * hiding this on enabling "with Call-To-Action(s)" isn't in line with what we do for fetching title/description from the page. _proposal_ disable it instead of hiding and move the "with Call-To-Action(s)" field below the link field.
* Description
  * The input background is always gray, even if the field isn't disabled (as a result of fetching it from the page). _expected:_ has a white background when it's enabled.

<img width="580" alt="screen shot 2018-05-23 at 14 25 57" src="https://user-images.githubusercontent.com/25616660/40424323-8813881c-5e95-11e8-9cee-4ad69b6f586d.png">